publish.sh

```diff
@@ -44,55 +47,86 @@ function run_provision_manager() {
 
 function build_gocli() {
   (cd cluster-provision/gocli && make container)
-  ${CRI_BIN} tag ${TARGET_REPO}/gocli ${TARGET_REPO}/gocli:${KUBEVIRTCI_TAG}
+  if [ $ARCH == "amd64" ]; then
+    ${CRI_BIN} tag ${TARGET_REPO}/gocli ${TARGET_REPO}/gocli:${KUBEVIRTCI_TAG}
+  else
+    ${CRI_BIN} tag ${TARGET_REPO}/gocli ${TARGET_REPO}/gocli:${KUBEVIRTCI_TAG}-${ARCH}
+  fi
 }
 
 function build_centos9_base_image_with_deps() {
   (cd cluster-provision/centos9 && ./build.sh)
   IMAGE_TO_BUILD="$(find cluster-provision/k8s/* -maxdepth 0 -type d -printf '%f\n' | head -1)"
+  if [ $ARCH == "s390x" ]; then
+    IMAGE_TO_BUILD="1.30"
```

Comment:
When will this be removed? After this PR has landed?

Reply:
I've removed this in [PR](https://github.com/kubevirt/kubevirtci/pull/1316) for 1.31 slim provider supporting s390x.